tests/harness/mod.rs

```diff
@@ -0,0 +1,211 @@
+use bitcoin::{util::psbt::PartiallySignedTransaction, Address, Amount};
+use bitcoin_harness::{bitcoind_rpc::PsbtBase64, Bitcoind};
+use reqwest::Url;
+use thor::{
+    create::{self, BuildFundingPSBT, Party6, SignFundingPSBT},
+    update,
+};
+use update::ChannelUpdate;
+
+pub struct CreateActors {
+    pub alice: create::Alice0,
+    pub alice_wallet: Wallet,
+    pub bob: create::Bob0,
+    pub bob_wallet: Wallet,
+}
+
+pub async fn make_create_actors(
```

Comment:
why not making it a `new` on `CreateActors` struct?

Reply:
No good reason! Will do.